src/modules/blockchainApplication/manage/components/ApplicationManagementRow/ApplicationManagementRow.test.js

```diff
@@ -25,15 +25,17 @@ useCurrentApplication.mockReturnValue([mockManagedApplications[1], mockSetApplic
 
 describe('ApplicationManangementRow', () => {
   let wrapper;
+  const location = {
```

Comment:
Neither location nor history is passed in as props to `ApplicationManagementRow`

Reply:
@oskarleonard Fixed